saplings/circuits/src/data/circuits.js

```diff
@@ -16,13 +16,39 @@
 
 import Paging from './paging';
 
+const argsToObject = coll => {
+  if (typeof coll === 'object') {
+    return Object.keys(coll).reduce((acc, k) => {
+      try {
+        acc[k] = JSON.parse(coll[k]);
+      } catch (e) {
+        acc[k] = coll[k];
+      }
+      return acc;
+    }, {});
+  }
+
+  if (Array.isArray(coll)) {
```

Comment:
I just realized, you should check for the array first. The fist if statement will be true for an array as well. The json for the scabbard arguments look like this for a proposal: 
<img width="645" alt="Screen Shot 2020-06-21 at 3 38 17 PM" src="https://user-images.githubusercontent.com/14094978/85234751-49748380-b3d5-11ea-8221-a3d0e53961bb.png">

Reply:
Good catch

Reply:
Fixed